civitai-downloader-v2/src/data/model_storage.py

```
@@ -0,0 +1,475 @@
+        def connection():
+            conn = None
+            try:
+                conn = sqlite3.connect(
+                    str(self.db_path),
+                    timeout=30.0,
+                    check_same_thread=False
+                )
+                conn.execute("PRAGMA foreign_keys = ON")
+                conn.row_factory = sqlite3.Row
+                yield conn
+            except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)